website/src/components/confetti/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useRef, useEffect } from 'react';
+import confetti from 'canvas-confetti';
+
+const ConfettiTrigger = ({ children }) => {
+  const triggerRef = useRef(null); // Use a ref to refer to the component's section
+
+  useEffect(() => {
+    const triggerConfetti = () => {
+        if (event.target.closest('a, h2, h3')) {
```

Comment:
`event` needs to be added in the parenthesis here: `const triggerConfetti = (event)`, otherwise `event.target` causes a TypeScript warning: `'event' is deprecated'

A good safety measure would also be to add optional chaining here: `event?.target?.closest`. This prevents the entire function from erroring out if `event` does not exist.

Currently, if event didn't exist, it would try to call `event.target` and cause an unhandled error, stopping the function at this point.